src/components/AdminMenu/index.tsx

```diff
@@ -1,11 +1,13 @@
 import { DollarSign, User, Image as ImageIcon, Circle } from 'react-feather'
+
```

Comment:
Can this component be moved to the admin layout?

Reply:
This might not be possible since in edit marketplace page we need to provide edit logo/banner functionality and for the same reason logo and banner images are not a part of the admin layout.

Reply:
@kespinola We can do one thing though. We can have 2 separate admin layouts - one for edit marketplace and other for edit creators/financials/tokens pages.

Reply:
ok.